docs/reference/faq-troubleshooting.md

```diff
@@ -1,17 +1,75 @@
+
+- hardlinks are a tricky thing for someone not familiar, but are worth understanding. They are a "direct line" to the actual data on the disk and are indistinguishable from the original to the file system. They do not require any extra space. These linktypes are only possible on the same partition/disk and you will need to have your linkDir set to the same mount (docker) or partition. This is the best approach if you do not keep torrents in your client (due to them being deleted from the tracker) - which would then break symlinks and cross-seeds. This is the approach commonly used in Arr's on import.
+
+### Searches aren't working from VPN'd torrent client
+
+If you are running your searches on completion from a VPN'd torrent client, it is likely you will need to set up split tunneling. You will not be able to access your local instance (which should not be publicly exposed) from the VPN due to localhost/NAT not being accessible from the VPN. If you are running a VPN'd torrent client in docker, you can use your cross-seed's container IP or name (if using custom docker networks.)
+
+### Searching media libraries vs. torrent data (data-based searching)
+
+You can search both your media libraries (Arr/Plex) and actual torrent data (downloaded files.) If you are using the media libraries, you will need to use `matchMode: "risky"` in your configuration file to allow cross-seed some leeway in its matching process for these renamed files. `risky` `matchMode` is not recommended to be used without skipRecheck set to false, as this is not as accurate as safe.
+
+- Due to the way data-based searching works, with tv media libraries cross-seed has no way of knowing whether the season is complete. Therefore it will not be able to match full seasons, even if the season is from a season pack.
+
+### My season packs are cross-seeding individual episodes!
+
+There is a configuration option, `includeSingleEpisodes`, which expands from `includeEpisodes`. If you wish to search for season pack's as a whole torrent and invidual episodes NOT from a season pack, you will need to set `includeEpisodes` to false, and `includeSingleEpisodes` to true. Both options would be best utilized with includeNonVideos set to true.
+
+### Why do I see "filetree is different" in my logs?
+
+This is due to the current matching algorithm cross-seed uses, and is most commonly associated with extra .nfo/.srt file's being present in a torrent or the possible case of one torrent being in a folder and the potential match not being (or vice versa.)
+
+### My tracker is mad at me for downloading 1000000 .torrent files!
+
+cross-seed currently searches for potential matches and compares the contents and size of files within, it is possible you are running searches too often, are not keeping the torrent_cache folder, or are improperly utilizing [**autobrr**](https://autobrr.com/).
```

Comment:
add a caveat that while we try to reduce grabs, to compare the files of two torrents we simply have to grab the torrent and some grabs are unavoidable